frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx

```diff
@@ -40,8 +38,7 @@ function QueryActionCardContent({ action }: { action: WritebackQueryAction }) {
 function ImplicitActionCardContent() {
   return (
     <ImplicitActionCardContentRoot>
-      <ImplicitActionIcon size={32} />
```

Comment:
Do we use this anywhere else?

Reply:
@kdoh, yeah, when opening a basic action in the editor

<img width="1066" alt="CleanShot 2023-02-23 at 11 49 48@2x" src="https://user-images.githubusercontent.com/17258145/220897837-de3eeda3-3538-4d1f-a21a-edd6276cb25d.png">